src/shared/Microsoft.Git.CredentialManager.Tests/EnumerableExtensionsTests.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.Git.CredentialManager.Tests
+{
+    public class EnumerableExtensionsTests
+    {
+        [Fact]
+        public void EnumerableExtensions_ConcatMany_Null_ThrowsException()
+        {
+            Assert.Throws<ArgumentNullException>(() => EnumerableExtensions.ConcatMany((IEnumerable<object>)null).ToArray());
+        }
+
+        [Fact]
+        public void EnumerableExtensions_ConcatMany_OneSequence_ReturnsSequence()
+        {
+            int[] seq1 = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
```

Comment:
You know I love unit tests :-).  Would it be worth throwing a dup number in the concated sequences for testing?

Reply:
Added duplicate values to tests.